pyk4a/pyk4a.py

```diff
@@ -72,38 +101,35 @@ def _stop_cameras(self):
         res = k4a_module.device_stop_cameras(self._device_id)
         self._verify_error(res)
 
-    def get_capture(self, timeout=TIMEOUT_WAIT_INFINITE, color_only=False, transform_depth_to_color=True):
-        r"""Fetch a capture from the device and return as numpy array(s) or None.
+    def get_capture(self, capture_request=CaptureRequest(), timeout=TIMEOUT_WAIT_INFINITE, ) -> CaptureResult:
```

Comment:
This function becomes much cleaner than before. Defaults are the same as before.